### ISSUE TITLE
Have the newscast tell you when it will have no effect?

### ISSUE THREAD

**Issue by peyre** [raceintospace/raceintospace#354]
Several of the newscasts don't affect your space program: the next mission must be manned, or must be a satellite, or Duration E must be completed before a Moon landing, etc.  So I've been wondering if it would be a good idea to add a notation to those newscasts.  We can't do much about the audio of course, but could add a little something to the end of the text.

I'm wondering if this is really such a good idea or if it's really unnecessary, and if it is a good idea what kind of addition I could make.  It would need to be short but still convey the basic idea.

**Comment by peyre** [comment]
A good example might be

IN GOVERNMENT NEWS TODAY, THE PRESIDENT HAS INSISTED THAT THE NEXT SPACE LAUNCH WILL BE A MANNED MISSION. HE HAS AUTHORIZED AN INCREASE OF 10 M.B.'S TO SUPPORT THIS EFFORT.

There isn't much room left in the hex file to add something, so it can't be very long.  Perhaps something like 
(NOT NECESSARY)
though that doesn't quite sit right with me.  I'd like to somehow very quickly say "Don't worry about this one."

**Comment by peyre** [comment]
Come to think of it, it doesn't have to be a cookie-cutter notation.  I'll compile a list here of these messages and see about coming up with something to add to the end of each.

**Comment by peyre** [comment]
These are the American announcements, since they're longer than the Soviet one - except the last is the Soviet message because its US equivalent doesn't sound like it affects you.  Suggested disclaimers are in bold.

SOURCES FROM THE SPACE PROGRAM HAVE DISCLOSED THAT DUE TO AN ENGINEERING ERROR, THERE IS A 50% CHANCE THAT THE NEXT MISSION WILL EXPLODE DURING LAUNCH.  **(THIS IS A FALSE NEGATIVE REPORTED BY THE ENGINEERS)**

MEDICAL DOCTORS AT THE UNITED STATES SPACE PROGRAM ANNOUNCED TODAY THAT AN ADDITIONAL 'DURATION MISSION E' WILL BE REQUIRED BEFORE ANY LUNAR LANDINGS WILL BE ATTEMPTED.  **(RECOMMENDED BUT NOT REQUIRED)**

IN GOVERNMENT NEWS TODAY, THE PRESIDENT HAS INSISTED THAT THE NEXT SPACE LAUNCH WILL BE A MANNED MISSION. HE HAS AUTHORIZED AN INCREASE OF 10 M.B.'S TO SUPPORT THIS EFFORT.  **(THE REQUEST DOES NOT HAVE TO BE HONORED)**

IN GOVERNMENT NEWS TODAY, THE PRESIDENT HAS INSISTED THAT THE NEXT SPACE LAUNCH WILL BE A MANNED MISSION. HE HAS AUTHORIZED AN INCREASE OF 10 M.B.'S TO SUPPORT THIS EFFORT.  **(THE REQUEST DOES NOT HAVE TO BE HONORED)**

THE GOVERNMENT ANNOUNCED TODAY THAT THE NSA AND THE PRESIDENT HAVE ORDERED THE SPACE AGENCY TO LAUNCH A SPY SATELLITE NEXT SEASON.  **(YOU CAN ACTUALLY LAUNCH WHENEVER IS CONVENIENT)**

COSMONAUT TRAINING HAS SUFFERED A SLOWDOWN DUE TO POWER OUTAGES AT FACILITIES.  **(NOT A SEVERE ENOUGH SLOWDOWN TO DELAY THEIR GRADUATION, HOWEVER)**

**Comment by rnyoakum** [comment]
Rather than modify the news text, I'm thinking the solution might be to change the display - in this case, the color. It might be reasonable to just flag all the news articles that don't have a game impact and render them in a different color, much like the standard seasonal news (the sports & culture announcements, etc.)

**Comment by peyre** [comment]
Could do that!  It seems almost the harder way around the issue though.

**Comment by rnyoakum** [comment]
Yeah, but changing the text to say "but this doesn't represent underlying game mechanics" breaks the immersion of the newscast.

**Comment by peyre** [comment]
That's quite true.  I haven't been able to think of a solution that gets the idea across without breaking the fourth wall, so to speak.

**Comment by mintdog58** [comment]
i think we should just leave that as part of the game to figure out ?????

**Comment by peyre** [comment]
It would be possible to do that, but shortage of money is meant to be one of the main challenges in the game.  Messing with it could kind of mess up the game.

I wonder, though, if we could have a setting in Advanced Preferences that would let you adjust the availability of funds.  Maybe something like `purse_strings x` where 1 would make money really tight and 5 would be very loose, with the default being 3.

**Comment by peyre** [comment]
Incidentally mintdog, there's now a cheat in the game.  In Purchasing, if you press $, you get 100MB.  You might find that fun to play with.

**Comment by peyre** [comment]
We could implement this idea sometime easily enough, if at some point news.dat is converted to a JSON.  Then we could add a flag for "Effect" (or something).  The values could be -1 for a newscast that hurts you, 1 if it hurts you, or 0 if it has no effect.  Or we could do 0,1,2 or whatever.

**Comment by peyre** [comment]
So, they've been converted to JSON!  We could do this now, except...event.json isn't following standard json format.  How do we co about adding a variable to it for this?
